src/foam/comics/BrowserView.js

```diff
@@ -78,13 +78,6 @@ foam.CLASS({
       name: 'selectEnabled',
       documentation: 'True to enable the select button.'
     },
-    {
-      class: 'Boolean',
-      name: 'addEnabled',
```

Comment:
Why is this removed?

Reply:
The 'add' action we have on the DAO controller is just a specific case of the more general 'select' action. Since we can use 'select' to select a bunch of objects to add or to remove, I think there's no need for a special action just for adding. That was my reason for removing it. If I'm missing some reason we need it though, please let me know.